osbs/build/build_request.py

```diff
@@ -728,7 +738,10 @@ def render(self, validate=True):
                           self.spec.koji_certs_secret.value,
 
                           ('prebuild_plugins', 'add_filesystem', 'koji_ssl_certs_dir'):
-                          self.spec.koji_certs_secret.value})
+                          self.spec.koji_certs_secret.value,
+
+                          ('postbuild_plugins', 'tag_and_push', 'secret'):
```

Comment:
Oh, I see, it's not added to the configuration for 'tag_and_push'. I think we should restructure that to make it more explicit, maybe by setting the argument name to None instead of 'secret', and skipping the dock_json_set_arg call in set_secret_for_plugin in that case.